src/app/components/navbar/navbar.component.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../../styles.scss';
+
+.navbar {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: #fff;
+    position: fixed;
+    width: 100%;
+    z-index: 999;
+}
+
+.navbar-brand {
+    @include flex-center;
+
+    input {
+        padding: 0.5rem 1rem;
+        background-color: #dce6f1;
```

Comment:
This color is defined at multiple places, kindly define the color and make it reusable